apps/aecore/lib/aecore/structures/account.ex

```diff
@@ -78,7 +78,11 @@ defmodule Aecore.Structures.Account do
   end
 
   def base58c_encode(bin) do
-    Bits.encode58c("ak", bin)
+    if bin == nil do
+      nil
```

Comment:
why do we need this? seems like a case for possible errors

Reply:
Yes, because sometimes when we serialize a coin-based transaction - the senders field will be nil as well as signature

Reply:
okay, didn't think of that